.gitignore

```diff
@@ -9,3 +9,4 @@ dist/index.js
 example/dist/
 example/package-lock.json
 
+/.pnpm-debug.log
```

Comment:
Please remove references to pnpm

Reply:
Is it okay to keep it in .gitignore so it won't be committed by accident?